### DISCUSSION TITLE
#資訊分享 Google Cloud AI Study Jam 2024 - 生成式 AI 培訓計劃

### DISCUSSION THREAD

**Discussion by irenesunnn** [1101-GS4719-C/Python-Community-GS4719-C#2151] [General]
Google Event Team近期的活動 ! [https://rsvp.withgoogle.com/events/csj-tw-2024]

Google Cloud 生成式 AI 培訓計劃（Cloud AI Study Jam） 將帶領參與者深入了解 Google Cloud 的生成式 AI 服務與技術。無論您是初學者還是具備經驗的開發者，透過三個不同難度的學習路徑，都可以透過課程學習與動手實作，掌握生成式 AI 的最新應用趨勢。

活動亮點：

深入學習 Google Cloud Platform 及其生成式 AI 工具
實際操作生成式 AI 應用，提升開發技能
完成指定的學習項目還能贏得 Google Cloud 獨家獎品！

10/31前完成所有路徑可以得到google背包跟保溫杯!!!

**Comment by Chattenoire** [comment]
感謝提供資訊!

**Comment by 112121py** [comment]
感謝版主提供的資訊，前陣子看到有去嘗試，真的還蠻受用的，而且隨著等級的不同，實作內容會越來越有料，推薦想嘗試或是想深入了解的人，除了保溫杯外也有很多不同的課程，而且不用錢，很佛r
